AdobeStockAsset/Model/Client.php

```diff
@@ -32,32 +32,35 @@ public function __construct(ConfigInterface $config)
     }
 
     /**
-     * @return mixed
+     * @param \Magento\AdobeStockAssetApi\Api\Data\RequestInterface $request
+     * @return array
+     * @throws \AdobeStock\Api\Exception\StockApi
      */
-    public function search()
+    public function execute(\Magento\AdobeStockAssetApi\Api\Data\RequestInterface $request)
     {
-        // TODO THIS IS A STUB
-        $results_columns = Constants::getResultColumns();
-        $search_params = new SearchParameters();
-        $search_params->setWords('tree')->setLimit(32)->setOffset(0);
+        // TODO: THIS IS A STUB. SHOULD BE REFACTORED
+        $words = $request->getData('filters')['words'] ?? 'image';
```

Comment:
Is "image" word the best query to use by default? Can we perform a request without words filter at all?

Reply:
We should define the search word because this field is required for request